configs/babel.config.js

```diff
@@ -7,6 +7,11 @@ module.exports = {
         [
             "@babel/env",
             {
+                "useBuiltIns": "usage",
+                "corejs": {
+                    "version": 3,
+                    "proposals": true
```

Comment:
`proposal` feature includes unstable feature.

- https://babeljs.io/docs/en/next/babel-preset-env.html#corejs

We will not support unstable feature by default.
Instead of it, user can add a polyfill by own.

Reply:
ok, but how? In fact I'm using this project to help development textlint, and I want to use `String.prototype.matchAll`, which is a will-coming feature in ES2020, but I don't know how to add polyfill for this. So I just install bebal as my own devDependendy and use my own bebal config, without using `textlint-scripts build`, and it's very unconvinent.